that says "the bibble"

Motoyuuri opened the pull request:
<!-- IT'S NOT WIZDENS REPO, IF YOU WANT TO ADD YOUR CHANGES ON ALL SERVERS, CREATE PR TO WIZDENS REPO -->

## Short description
Adds a new holy book, the bibble, accessible by cutting the manager wire on the pietyvend. Also now splits off the starlight specific holy books into their own .yml file to smooth out integration of any upstream bible changes!

## Why we need to add this
Very minor comedic change, adds an additional easter egg item for the chaplain, cleans up resource organization a bit, makes further changes easier to merge

## Media (Video/Screenshots)
<img width="481" height="403" alt="image" src="https://github.com/user-attachments/assets/4a7a8fdc-571a-48d5-899c-6dd312239e50" />
<img width="420" height="182" alt="image" src="https://github.com/user-attachments/assets/898aafc1-a6f8-47c1-a89d-d2d16519e9d1" />
<img width="954" height="636" alt="image" src="https://github.com/user-attachments/assets/b360c555-4924-4f1b-b359-0a08e8c9f03d" />



## Checks
<!-- check boxes for faster reviewing of your PR -->

- [x] I do not require assistance to complete the PR.
- [x] Before posting/requesting review of a PR, I have verified that the changes work.
- [x] I have added screenshots/videos of the changes, or this PR does not change in-game mechanics.
- [x] I affirm that my changes are licensed under the [Starlight Fork License](https://github.com/ss14Starlight/space-station-14/blob/Starlight/LICENSE-Starlight.TXT) and grant permission for use in this repository under its conditions.

**Changelog**
<!--
If you want the players to know about changes made in this PR, specify them using the template outside the comment. Short and informative.

:cl: motoyuuri
- add: textures and code for the bibble
- tweak: updates pietyvend inventory to add the bibble to the manager wire
-->
:cl: motoyuuri
- add: textures and code for the bibble
- tweak: updates pietyvend inventory to add the bibble to the manager wire

Motoyuuri committed:
Add files via upload

Adds an rsi file containing textures for the bibble

Motoyuuri committed:
Update chapel.yml

Adds 1 copy of the bibble to contrabandInventory, behind the manager wire

Motoyuuri committed:
Update bibles.yml

adds the bibble to bibles.yml

Motoyuuri committed:
Update bibles.yml

Motoyuuri committed:
Rename inhand_left.png to inhand-left.png

Motoyuuri committed:
Rename inhand_right.png to inhand-right.png

Penguinwizzard commented on Resources/Prototypes/Catalog/VendingMachines/Inventories/chapel.yml:
```diff
@@ -32,6 +32,7 @@
     DrinkWineBottleFull: 1
     ToyFigurineChaplain: 1
     ClothingNeckScarfStripedBlack: 3
+    BibleBibble:  1
```
Needs a `# Starlight` comment

Penguinwizzard commented on Resources/Prototypes/Entities/Objects/Specific/Chapel/bibles.yml:
```diff
@@ -353,5 +353,15 @@
   - type: Item
     sprite: _Starlight/Objects/Specific/Chapel/rigveda.rsi
-
+    
```
Can you add this in `Resources/Prototypes/_StarLight/Entities/Objects/Specific/Chapel/bibles.yml` instead? It'd help avoid the possibility of merge conflicts from this.

Motoyuuri replied:
this doesn't seem to currently be a path in the sl code? all the current sl bible variants are in the normal bibles.yml file - I could try changing this but it'd require a lot of fixes, mainly in the chaplain starting inventory files

Motoyuuri committed:
Update chapel.yml

added a #starlight comment for the bibble in the contrabandinventory section of the pietyvend

Motoyuuri reviewed (commented):
Fixed the missing #starlight comment in inventories/chapel.yml

Motoyuuri committed:
Create Chapel in _Starlight/entities/objects/specific

Motoyuuri committed:
Delete Resources/Prototypes/_StarLight/Entities/Objects/Specific/Chapel

Motoyuuri committed:
Merge branch 'starlight-dev' into newbook

Motoyuuri committed:
Add _starlight/entities/objects/specific/chapel/bibles.yml

splits off the starlight specific bibles into their own .yml file to smooth out any upstream changes

Motoyuuri committed:
Update bibles.yml

removes the starlight specific bibles from bibles.yml; they're now in their own .yml in the _starlight entities path

Motoyuuri commented:
nevermind actually! turns out creating a new bibles.yml under the _starlight path was easy and didn't cause any problems at all - have comitted additional changes to the pull req that splits the starlight-specific holy books into their own file, which should make this pull req not only add the new book but also prevent any issues when merging future upstream bible updates

Penguinwizzard reviewed (approved):
Thanks for moving the files, LGTM!

Happyrobot33 committed:
Merge branch 'starlight-dev' into newbook

Happyrobot33 committed:
Update meta.json

Happyrobot33 committed:
newlines fix?

Happyrobot33 committed:
Update metadata for bibble.rsi

Happyrobot33 commented:
trying to fix the crlf myself

Commit:
fix crlf
